addons/survey/views/survey_templates.xml

```diff
@@ -300,6 +300,12 @@
         </div>
     </template>
 
+    <template id="question_answer_image" name="Question answer: image">
```

Comment:
I would rename `question_suggested_answer_image` to avoid confusion as it has nothing to do with the "answer" coming from the person taking the survey.
And `name="Image from the question suggested answer"`.

Reply:
done